### PR TITLE
🐛 Source Github: Remove optional parameter `Accept` for reaction's streams to fix error with `502` HTTP status code

### PR DESCRIPTION
**Pull request by yevhenii-ldv**
## What
*This PR is intended to fix the problem that arises in the https://github.com/airbytehq/oncall/issues/75 task. Unfortunately, I was never able to reproduce the error that appears in the user, however, I found an article on the [Github forum](https://github.community/t/api-request-returning-502-persistently/13551) that 502 errors sometimes appear due to the `Assert` header, so I removed it, after checking the data for identity with and without this header him..*

## How
*Remove `Accept` header for Reaction's streams*

## Recommended reading order
1. `x.java`
2. `y.python`

#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

---
This change is [Reviewable](https://reviewable.io/reviews/airbytehq/airbyte/9492)

### CODE REVIEW

**Review by htrueman — APPROVED** [review]

File: airbyte-integrations/connectors/source-github/source_github/streams.py [hunk 1]
```diff
@@ -404,7 +409,7 @@ def read_records(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Iter
         yield from super().read_records(stream_state=stream_state, **kwargs)
 
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
-        return f"repos/{stream_slice['repository']}/pulls"
+        return f"repos/{stream_slice['repository']}/pullsasd"
```
Comment (Zirochkaa) [inline airbyte-integrations/connectors/source-github/source_github/streams.py:412]: Is it called `pullsasd` right now?

**Comment by Zirochkaa** [comment]
Please use template from `.github/PULL_REQUEST_TEMPLATE/bug-fix_template.md` file for PR description because this is a bug fix.

**Review by Zirochkaa — APPROVED** [review]

**Pull request merged**